docs/developer-docs/branding-guide.md

```diff
@@ -0,0 +1,34 @@
+# Jellyfin Branding Guide
+
+## Jellyfin UX Repository
+All Jellyfin UI/UX resources are kept in the jellyfin-ux repository which can be found on github at https://github.com/jellyfin/jellyfin-ux
+
+This repository contains:
+
+* SVG files for icons
+* SVG files for banners
+* PNG files of icons and banners
+* Fonts
+
+
+## Jellyfin Colours
+
+These are the colours used for the Jellyfin Logos and UI
+
+### Logo Colours
+
+The design for the logo uses a gradient for the infill, and if the non transparent logo is chosen there is a background colour.
+
+* Gradient Start: #AA5CC3 
+* Gradient End: #00A4DC
+* Background Colour (optional): #000B25 
+
+### UI Colours
+
+* Background Colour: 
```

Comment:
Why it's empty?

Reply:
The UI colours? There was still an ongoing conversation in riot about UI colours and accent colours while I was writing this